### PR TITLE
Remove headers, update log

### PR DESCRIPTION
**Pull request by Aareksio**
Remove headers from pages, format log, refactor some elements into own components.

### CODE REVIEW

**Comment by MrBurrBurr** [comment]
In UI Configuration you forgot to remove header. Also `Default page` setting needs to be updated.

**Comment by Aareksio** [comment]
1) It is intended. These _headers_ are inside _container_ component. I removed only these outside.
2) Gonna fix this
3) That's right. As in comment I wrote earlier (can't find it anymore) - that was the first and only reason it wasn't implemented before. I don't think it's such a big deal though, let's be honest - how many people actually customize the history log format? We may consider adding a configuration in settings if it becomes a problem.
4) We may remove it on smaller screens if there's not enough space. I don't think changelog link hurts anyone otherwise.
5) I was against this since the very begining. `Cards remaining: 0` is still valuable information - especially for first time users.
6) Where exactly?

**Comment by JustArchi** [comment]
The right solution to your logging problem looks like exposing information about `HistoryTarget`, maybe as part of `/Api/ASF`, the question is if you're interested in that.

**Comment by MrBurrBurr** [comment]
2. Just curious: Any way to get navigation links dynamically?
3. IMHO we should make use of what Archi suggested
5. But if no bot is farming we can't know how many cards are remaining. Maybe I am getting something wrong here but wouldn't that be misleading information?
6. Oh just in the code itself

**Comment by Aareksio** [comment]
2. Yes, [documentation](https://router.vuejs.org/api/#router-resolve)
3. Already coded basic parser, but it requires significant amount of hardcoded information I'd like to avoid. Discussing in #programming on Discord.
4. (5) If that's the case, then we're misleading anyway whenever *some* of the bots are not farming. I don't really understand how one is _less_ misleading than another. If anything, I'd assume a value "0" is suggesting "(some) data unavailable" rather than "random_number".
5. (6) Good catch. As I auto extracted components with IDE the spacing had to get corrupted, gonna fix.

**Comment by MrBurrBurr** [comment]
5. I don't think we are misleading the user if we hide farming information until we are actually farming. Once we are farming, the displayed farming information will be correct.

**Comment by Aareksio** [comment]
It'll account only for the bots that are in _farming_ state, right? If so, the information is equally misleading as long as _some_ bots are not farming.

I am against any of such layout parts configuration, as it makes us maintain more and more possible UI configuration with time. If anything, replacing '0' with '-' when no bots are farming is an option.

**Comment by MrBurrBurr** [comment]
> If anything, replacing '0' with '-' when no bots are farming is an option.

👍

**Comment by Aareksio** [comment]
`home` is an alias to whatever page we select as the main one, I'd like to leave it as a default in case we decide to switch it later on, not to break the behavior for average user.

Gonna add `changelog` to the list, although I can't imagine anyone intentionally using it 🙂

File: src/components/BotFarmingInfo.vue [hunk 1]
```diff
@@ -1,61 +1,39 @@
...
 
 				const language = ['zh-CN', 'zh-TW'].includes(this.$i18n.locale)
 						? this.$i18n.locale.replace('-', '_')
 						: this.$i18n.noRegionalLocale;
 
 				return humanizeDuration(this.$store.getters['bots/timeRemaining'] * 1000, { language });
+			},
+			gamesRemaining() {
+				if (this.botsFarmingCount === 0) return '-';
+				return this.$store.getters['bots/timeRemaining'];
```
Comment (MrBurrBurr) [inline src/components/BotFarmingInfo.vue]: This should be fixed.

**Pull request merged**